packages/pluggableWidgets/line-chart-native/src/LineChart.xml

```diff
@@ -42,12 +42,12 @@
                                 </attributeTypes>
                             </property>
                             <property key="staticSeriesName" type="textTemplate" required="false" multiline="false">
```

Comment:
I see that we renamed series to line but kept the internal properties the same name. This would be fine IF you wouldn't name the styles to `staticLineStyle` 

Dont you wanna rename the properties to `staticLineName` and `dynamicLineName` ?

Reply:
Refactored it.